actors/builtin/sector.go

```diff
@@ -7,43 +7,47 @@ import (
 
 // Policy values associated with a seal proof type.
 type SealProofPolicy struct {
-	SectorMaxLifetime      stabi.ChainEpoch
+	SectorMaxLifetime stabi.ChainEpoch
 }
 
-// For all Stacked DRG sectors, the max is 5 years
+// For V1 Stacked DRG sectors, the max is 540 days
+// 	according to https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0014.md
+const EpochsIn540Days = stabi.ChainEpoch(540 * EpochsInDay)
+
+// For V1_1 Stacked DRG sectors, the max is 5 years
 const EpochsInFiveYears = stabi.ChainEpoch(5 * EpochsInYear)
 
 var SealProofPolicies = map[stabi.RegisteredSealProof]*SealProofPolicy{
 	stabi.RegisteredSealProof_StackedDrg2KiBV1: {
-		SectorMaxLifetime:      EpochsInFiveYears,
+		SectorMaxLifetime: EpochsIn540Days,
```

Comment:
We *might* be able to get away with this.  Without looking into this in depth I'm concerned that SectorMaxLifetime of v1 seal proofs somehow made its way into the code path of network version 10 v3 actors (current mainnet).  If that's the case this change could cause different behavior and prevent this code from syncing version 10 state transitions.  

It might be the case that the current extension ban in v10 has prevented this field from ever being read by network version 10 actors code, in which case this will work.  I will dig in deeper and provide a more definitive suggestion soon.

Reply:
I agree this is probably fine, but I also agree that it's safer and easier to reason about to just have two maps and add a function that takes the current network version and returns the policy. I'd prefer the latter approach.

Reply:
I concur

Reply:
@steven004 this comment is not yet resolved.  The recommendation after discussion here and on slack is that we have two seal proof policy maps and use a function taking the network version to switch between them whereever we read this.

Reply:
Sorry for missing this. Now it should be solved. please check.